misc/documentation_styleguide.md

```diff
@@ -53,6 +53,24 @@ This is some text.
 This is some more text.
 ```
 
+## Bullet and numbered items
+
+First level is flushed all the way to the left. A sub level is offset by 4 spaces. The next sub level is offset by 8 spaces. You get the idea!
```

Comment:
While this definitely makes sense, let's put it a bit more technical ;-)

```
The first level has no indentation, and it's completely aligned to the left. Sub levels should be indented using 4 spaces. Example:
```

Also add a note about using `*` instead of `-` to create items.